### PR TITLE
Disable update check through an environment variable

### PR DESCRIPTION
**Pull request by monkbroc**
Allow the CLI installer to start the CLI without the update check.

### CODE REVIEW

File: settings.js [hunk 1]
```diff
@@ -100,6 +100,22 @@ var settings = {
...
 };
 
+function envValue(varName, defaultValue) {
+	var value = process.env[varName];
+	return (typeof value === 'undefined') ? defaultValue : value;
+}
+
+function envValueBoolean(varName, defaultValue) {
+	var value = envValue(varName);
+	if (value === 'true' || value === 'TRUE' || value === '1') {
```
Comment (stefan-moraru) [inline settings.js]: Are we sure we aren't going to have integer 1s?

File: settings.js [hunk 2]
```diff
@@ -100,6 +100,22 @@ var settings = {
 	commandMappings: path.join(__dirname, 'mappings.json')
 };
 
+function envValue(varName, defaultValue) {
+	var value = process.env[varName];
+	return (typeof value === 'undefined') ? defaultValue : value;
+}
+
+function envValueBoolean(varName, defaultValue) {
+	var value = envValue(varName);
```
Comment (stefan-moraru) [inline settings.js]: We should pass the defaultValue to 'envValue'

Comment (monkbroc) [inline settings.js]: No, this is on purpose to get the raw value. The default handing is in `envValueBoolean`

File: settings.js (hunk 1 above)
Comment (monkbroc) [inline settings.js]: Environment variables are always strings.

**Pull request merged**